src/views/certifications/views/[slug]/components/heading-permalink/index.tsx

```diff
@@ -0,0 +1,17 @@
+import slugify from 'slugify'
+import { IconLink16 } from '@hashicorp/flight-icons/svg-react/link-16'
+import s from './heading-permalink.module.css'
+
+export function HeadingPermalink({ heading }: { heading: string }) {
+	const id = `${slugify(heading, {
+		lower: true,
+	})}`
+	return (
+		<h2 className={s.heading} id={id}>
+			<a className={s.anchorLink} href={`#${id}`}>
```

Comment:
I see the current anchor links plugin as an aria-label on the anchor. If we did that here it would read `${heading} permalink`. Would that be helpful? 💭

Reply:
I don't think we need it here since the interactivity of this element isn't communicated until someone hovers over or focuses on this link. We'd also not want to include `"permalink"` for the voice control experience, since the Accessible Name of an element (can be set by `aria-label`) needs to match the visible label. The Accessible Name is already the heading text based on how that's calculated by browsers, so I think we're good here!